commune_app/all_models/users.py

```diff
@@ -38,10 +38,12 @@ def save(self, *args, **kwargs):
         return super().save(*args, **kwargs)
 
     def join_commune(self, commune_id):
+        commune = Commune.objects.filter(id=commune_id).first()
         if (self.commune_id is not None and self.commune_id != commune_id):
-            raise Exception("already partner in another commune")
+            raise Exception("already partner in the commune: " + self.commune_id.name)
         self.commune_id_id = commune_id
         self.save()
+        return "you joined to commune: " + commune.name
```

Comment:
could you please elaborate on why do we need to return this string?

Reply:
I tried to understand what you meant by using the name of the commune, that's the only thing that occurred to me, maybe we would like to return such a message to the user